content/02.introduction.md

```diff
@@ -90,12 +90,16 @@ Coronaviruses are known to cause respiratory illnesses in humans through the fol
 
 #### Clinical Presentation of COVID-19
 
-Retrospective samples of COVID-19 patients described the clinical presentations of patients infected with SARS-CoV-2 which included lower respiratory tract infection with fever, dry cough, and dyspnea [@doi:10/ggnxb3].
+A great diversity of symptom profiles has been observed for COVID-19, although fever and cough are very common.  [@doi:10.1056/NEJMoa2002032]
+One early retrospective study in China described the clinical presentations of patients infected with SARS-CoV-2 which included lower respiratory tract infection with fever, dry cough, and dyspnea [@doi:10/ggnxb3]. 
 [@doi:10/ggnxb3] noted that upper respiratory tract symptoms were less common, which suggests that the virus targets cells located in the lower respiratory tract.
 The symptoms of infection by SARS-CoV-2 can vary greatly, making it difficult for public health agencies to provide clear recommendations for citizens regarding what symptoms indicate infection and should prompt isolation.
-[@doi:10/ggnxb3] found that a higher probability of mortality was associated with older age and higher Sequential Organ Failure Assessment scores, as well as high levels of d-dimer.
-Mortality might be associated with other biomarkers measured in blood samples including lactate dehydrogenase and cardiac troponin I, although these analyses may not have been appropriately corrected for multiple testing.
-They also found that survivors continued to shed the virus for a median of 20 days and a maximum of at least 37 days.
+[@doi:10/ggnxb3] found that a higher probability of mortality was associated with older age and higher Sequential Organ Failure Assessment scores, as well as high levels of d-dimer. 
+Mortality might be associated with other biomarkers measured in blood samples including lactate dehydrogenase and cardiac troponin I, although these analyses may not have been appropriately corrected for multiple testing. 
+[@doi:10/ggnxb3] reported that survivors continued to shed the virus for a median of 20 days and a maximum of at least 37 days.
+A later study reported that many hospitalized patients had radiographic findings such as ground-glass opacity and bilateral patchy shadowing, and most had lymphocytopenia. [@doi:10.1056/NEJMoa2002032]
+COVID-19 can affect diverse body systems in addition to causing respiratory problems.  
+For example, COVID-19 can lead to acute kidney injury, especially in patients with severe respiratory symptoms or certain preexisting conditions [@doi:10.1016/j.kint.2020.05.006].
```

Comment:
```suggestion
For example, COVID-19 can lead to acute kidney injury, especially in patients with severe respiratory symptoms or certain preexisting conditions [@doi:10.1016/j.kint.2020.05.006].  COVID-19 has also been associated with an increased incidence of large vessel stroke, particularly in patients under the age of 40 [@doi:10.1056/NEJMc2009787], and other thrombotic events including pulmonary embolism and deep vein thrombosis [@doi:10.1016/j.thromres.2020.04.013].  The mechanism behind these complications has been suggested to be related to coagulopathy with reports of antiphospholipid antidbodies present [@doi:10.1056/NEJMc2007575] and elevated levels of d-dimer and fibrinogen degradation products (FDP) elevated in deceased patients [@doi:10.1111/jth.14768]. 
```

Reply:
I am so glad that you are expanding this section!  I am not a medical expert and was just summarizing studies I had read.  I would defer to @rando2 but I think it would be great to be as detailed as possible.  
We also didn't say anything yet about pediatric symptoms and MIS/C or Kawasaki-like syndrome yet, as far as I know -- I feel like somebody should say something about that.  I could work on a rough outline but I am sure that @scapone01 would be much better qualified to discuss it intelligently.

Reply:
@dziakj1 @scapone01 this seems like a good level of detail to include to me! I know both renal failure and antiphospholipid syndrome have at various point been factored into considerations for why HCQ might or might not be useful for COVID-19, so this is very interesting to see here as well. I would actually say, if you wouldn't mind explaining coagulopathy a little more and whether it is seen in other viral infections, that would also be interesting background!

Reply:
@rando2 @dziakj1 Sounds good, I'll get to work on this.  I'll provide a little extra background, but don't want to send people down too much of a rabbit hole.  I'll also add in some more detail on both presenting symptoms and outcomes.  I'm going to do my best to not inundate you two with updates and do it in an efficient manner.  I apologize in advance, I'm a Github rookie.

Reply:
@scapone01 that sounds great! Please let me know if you need any help, GitHub is a new platform for most people working on this! I tried to put together some info [here](https://github.com/greenelab/covid19-review/blob/master/CONTRIBUTING.md#reviewing-someones-pull-request) that might help

Reply:
@rando2 Thank you, that page has been incredibly helpful.  I updated my comment here to add some additional information about viral infections and coagulopathy.  I can go into more detail about the specifics of the coagulation cascade if you think that's relevant.

Reply:
@scapone01 this is super interesting!!! I made some superficial edits (just split the lines apart). @dziakj1 does this look good to you?

Reply:
@scapone01  There were a few places where I wasn't sure what you meant.  Could you be clearer about "were initially reported to be the two most common symptoms on admission"?  I don't know what "initially" means here, because I'm not confident about whether the article we are citing is the very first article ever on COVID-19.  Also, " on room air" might not be obvious to all readers;  could it be "when not on artificial oxygen supplementation or ventilation"?  I also don't know what you mean by "variable rates of fever as a presenting symptom."  Do you mean that there are different rates at different times, or different places, or for different people?  The context of the sentence suggests that only New York is being discussed in that place, so I don't know what is supposed to vary.  Thanks!

Reply:
Yea this one needs some work.  All very good points.  I'll edit tonight and try to clarify each of those points and make it more concise and cleaned up.

Reply:
@dziakj1 How do you feel about the other additions regarding complications?